### ISSUE TITLE
Cannot add a JPG image into exported Excel

### ISSUE THREAD

**Issue by YuerLee** [ag-grid/ag-grid#8634]
**I'm submitting a ...** 
```
[x] bug report => see 'Providing a Reproducible Scenario'
[ ] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[ ] support request => see 'Requesting Community Support'
```

**Current behavior**
Starting from AG Grid v31.3.0, a JPG image converted to base64 cannot be added to the exported Excel cell.
- v31.2.0 ⭕ https://plnkr.co/edit/Rr8ScQi3FUrqe91A?open=main.ts&preview
- v31.3.0 ❌ https://plnkr.co/edit/KzIMXCGsGJfa94GY?open=main.ts&preview
- v32.1.0 ❌ https://plnkr.co/edit/XwqAh5R6cRpAMLvi?open=main.ts&preview

**Expected behavior**
Can add a JPG image into the exported Excel.

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference: **AG-12963**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can follow the status of this item on the product pipeline page:
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil

**Comment by YuerLee** [comment]
Fixed in v32.3.0.
